server/stachehelper.js

```diff
@@ -6,24 +6,91 @@ var moment      = require("moment")
 var path        = require("path")
 var marked      = require("marked")
 var fs          = require("fs")
-var highlight   = require("node-syntaxhighlighter")
+var highlight   = require("highlight.js")
 var RSS         = require("rss")
 var config      = require("../config")
+var logger      = require("./logger");
+
+var renderer    = new marked.Renderer()
+
+var langs = {
+	"js": "JavaScript",
+	"ts": "TypeScript",
+	"bash": "Bash",
+	"json": "JSON"
+}
 
 marked.setOptions({
 	gfm: true,
 	highlight: function(code, lang){
-		return highlight.highlight(code, highlight.getLanguage(lang ? lang : "text"))
+		if (lang !== undefined) {
+			let highlighted = highlight.highlight(lang, code).value
+
+			if (lang === "bash") {
```

Comment:
Not without modifying highlight.js source, which would get overwritten every time we update it.